gtsfm/utils/verification.py

```diff
@@ -74,38 +79,65 @@ def essential_to_fundamental_matrix(
     return np.linalg.inv(camera_intrinsics_i2.K().T) @ i2Ei1.matrix() @ np.linalg.inv(camera_intrinsics_i1.K())
 
 
-def compute_epipolar_distances(
-    normalized_coords_i1: np.ndarray, normalized_coords_i2: np.ndarray, i2Ei1: EssentialMatrix
+def compute_epipolar_distances_sed(
+    coordinates_i1: np.ndarray, coordinates_i2: np.ndarray, i2Fi1: np.ndarray
 ) -> Optional[np.ndarray]:
-    """Compute symmetric point-line epipolar distances between normalized coordinates of correspondences.
+    """Compute symmetric point-line epipolar distance between corresponding coordinates in two images. The SED is the
+    geometric point-line distance and is a biased estimate of the gold-standard reprojection error.
+
+    References: 
+    - "Fathy et al., Fundamental Matrix Estimation: A Study of Error Criteria"
```

Comment:
maybe URL and year would make easier to find?

Reply:
can we put simple math equation in the docstring?

Reply:
better?